README.md

```diff
@@ -1,44 +1,48 @@
 <a name="readme-top"></a>
 
 [![MIT License][license-shield]][license-url]
-[![Download in the Chrome Web Store][chrome-shield]][chrome-url]
-[![Download in the Firefox Add-on Store][firefox-shield]][firefox-url]
 
 <!-- PROJECT LOGO -->
-<br />
+
 <div align="center">
-  <a href="https://github.com/AndrewWalsh/openapi-devtools">
-    <img src="resources/logo.svg" alt="Open API dev tools" width="300" height="250">
+  <a href="https://github.com/Superflows-AI/openapi-llm">
+    <img src="resources/logo.svg" alt="LLM for OpenAPI DevTool" width="300" height="250">
   </a>
+</div>
 
+<br />
 
-  <p align="center" style="max-width: 600px;">
-    Effortlessly discover API behaviour with a browser extension that automatically generates OpenAPI specifications in real time for any app or website.
-    <br />
-    <br />
-    <a href="https://awalsh.io/posts/openapi-devtools/">Read More</a>
-    ·
-    <a href="https://github.com/AndrewWalsh/openapi-devtools/issues">Report Bug</a>
-    ·
-    <a href="https://forms.gle/fA2EtvamUrsRxXKYA">Give Feedback</a>
-  </p>
-  </p>
-</div>
+ Generate a OpenAPI Specification in real time for any app or website, including descriptions of endpoints and parameters.
+
+<br />
+
+A browser extension that discovers API behaviour via your interactions with a product or website, and then uses an LLM to describe the API endpoints discovered.
+
+<br />
+
+This project is a fork of OpenAPI Devtools. Check out the <a href="https://github.com/AndrewWalsh/openapi-devtools"> original tool here. </a>
+
+<br />
+
+The <a href="https://www.superflows.ai/"> Superflows </a> team extended the functionality of the original tool after building AI assistants using OpenAPI specifications. We found it useful to have natural language descriptions in OpenAPI Specifications, both for human understanding and to increasing the reliability of AI assistants using them to make API calls. 
 
 ## About The Project
 
 <p align="center" width="100%">
     <img width="80%" src="resources/demo.gif">
 </p>
 
-OpenAPI DevTools is a browser extension that generates OpenAPI specifications in real time from network requests. Once installed it adds a new tab to DevTools called `OpenAPI`. While the tool is open it automatically converts network requests into a specification.
+<p>
+OpenAPI-LLM is built on top of the OpenAPI DevTools browser extension that generates OpenAPI specifications in real time from network requests. Once installed it adds a new tab to DevTools called `OpenAPI`. While the tool is running, it automatically converts network requests into a specification. You can then select endpoints you would like to have a natural language descriptions for, and generate descriptions of endpoints and parameters. 
+
+We customised this for the use case of feeding these OpenAPI Specifications to LLMs or AI agents. LLMs and AI typically require thorough descriptions of endpoints to use them well. 
```

Comment:
Potentially should link to [this section](https://docs.anthropic.com/claude/docs/tool-use#best-practices-for-tool-definitions) to make the point stronger that descriptions are important for LLM -> API?